scalardl-test/src/main/java/scalardl/transfer/TransferChecker.java

```diff
@@ -27,12 +27,23 @@
 
 public class TransferChecker extends PostProcessor {
 
+  private final int SLEEP_TIME_MILLISECONDS = 60000;
+
   public TransferChecker(Config config) {
     super(config);
   }
 
   @Override
   public void execute() {
+
+    try {
+      Thread.sleep(SLEEP_TIME_MILLISECONDS);
```

Comment:
It's the Jepsen-specific issue, so I don't think we should add sleep here. It's a general test.
Can you add sleep in the script or the jepsen code?

Reply:
Sorry I should have provided more detail in the description.
The issue is occurring in periodic scalar DL verification test in post check phase of kelpie test.  
I don't think Jepsen code is used in this specific test.

Reply:
Ah, OK.
Anyway, sleeping 60 seconds all the time is not good because it's only needed when underlying databases are down so it should be at least configurable. (default: 0)

Reply:
Understood, I will change it and make it configurable.